PayPal/src/main/java/com/braintreepayments/api/PayPalCheckoutRequest.java

```diff
@@ -0,0 +1,236 @@
+package com.braintreepayments.api;
+
+import android.text.TextUtils;
+
+import androidx.annotation.StringDef;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+
+public class PayPalCheckoutRequest extends PayPalRequest {
```

Comment:
I think we should add a doc string here.

Reply:
Added 👍